docs/pages/management/admin/labels.mdx

```diff
@@ -1,405 +1,323 @@
+
+1. Add a `command` array that runs the `uname` command with the `-p` argument to return the architecture of the 
+host server every one hour. 
+   
+   For example, add the `name`, `command`, and `period` fields as follows:
+
+   ```yaml
+   ...
+   ssh_service:
+     enabled: "yes"
+     labels:
+       teleport.internal/resource-id: 1f2cdcc5-cde3-41fa-b390-bc872087821a
+       environment: dev
+     commands:
+     - name: hostname
+       command: [hostname]
+       period: 1m0s
+     - name: arch
+       command: [uname, -p]
+       period: 1h0m0s
+   ```
+   
+   In the `command` setting, the first element is a valid executable. 
+   Each subsequent element is an argument.
+   The following syntax is valid:
+
+   ```yaml
+   command: ["/bin/uname", "-m"]
+   ```
+   
+   The following syntax is not valid:
+
+   ```yaml
+   command: ["/bin/uname -m"]
+   ```
+   
+   For more complex commands, you can use single (') and double (") 
+   quotation marks interchangeably to create nested expressions. 
+   For example:
+
+   ```yaml
+   command: ["/bin/sh", "-c", "uname -a | egrep -o '[0-9]+\\.[0-9]+\\.[0-9]+'"]
+   ```
+   
+   In configuring commands, keep the following in mind:
+   
+   - You must specify the path to a valid executable program or file. 
+   - You must set the executable permission bit on any file you use as a command. 
+   - Shell scripts must have a [shebang line](https://en.wikipedia.org/wiki/Shebang_\(Unix\)).
+   
+   The `period` setting specifies how frequently Teleport executes each command.
+   In this example, the `uname -p` command is executed every one hour (1h), zero minutes (0m),
+   and zero seconds (0s). This value can't be less than one minute.
+
+1. Save your changes and close the file.
+
+1. Start Teleport with the invitation token you saved in the `INVITE_TOKEN` environment variable:
+
+   ```code
+   $ sudo teleport start --token=${INVITE_TOKEN?}
+   ```
+
+1. Verify that you have added the label by running the following command on your local
+computer. Your Teleport user must be authorized to access the Node.
+
+   ```code
+   $ tsh ls
+   ```
+
+   You should see output similar to the following with both the `arch` and `environment` labels displayed:
+
+   ```code
+   Node Name        Address        Labels                                                                                                                   
+   ---------------- -------------- ------------------------------------------------------------------------------------
+   ip-192-168-13-57 ⟵ Tunnel       arch=x86_64,environment=dev,hostname=ip-192-168-13-57,teleport.internal/resource-id 
+   ```
+
+## Adding resources and restarting
+
+In this guide, you learned how to add and apply a static label and a command-based dynamic label 
+for a server resource. If you want to add labels for other resources—for example, a database 
+or application—you might have to create another invitation token using the `tctl tokens add` command
```

Comment:
I feel like this is unnecessarily coupling adding resources to setting labels. I suspect a user looking to learn how to add resources probably would have landed on a different docs page, and most readers of this page will be more interested in adding labels to existing resources. (In which case the invitation token is unnecessary and just complicates this guide)

Reply:
I think it makes sense to drop this section completely. What was there before didn't seem very useful, but trying to morph it into something else didn't work either.